File: DS-Algo-with-Java/Lecture-3/LogicalOperators.java
public class LogicalOperators {
    public static void main(String[] args) {
        int p = 15, q = 10, r = 5;

        // && Operator
        System.out.println((p > q) && (p > r)); // True
        System.out.println((p > q) && (p < r)); // False

        // || Operator
        System.out.println((r < q) || (p < q)); // True
        System.out.println((p > q) || (q > r)); // True
        System.out.println((p < q) || (q < r)); // False

        // ! Operator
        System.out.println(!(p == q));          // True
        System.out.println(!(p > q));           // False
    }
}

File: DS-Algo-with-Python/Lecture-3/LogicalOperators.py
p = 15
q = 10
r = 5

# AND Operator
print((p > q) and (p > r))  # True
print((p > q) and (p < r))  # False

# OR Operator
print((r < q) or (p < q))   # True
print((p > q) or (q > r))   # True
print((p < q) or (q < r))   # False

# NOT Operator
print(not(p == q))          # True
print(not p > q)            # False
